packages/ui/src/IconButton/IconButton.tsx

```diff
@@ -13,6 +15,7 @@ export enum IconButtonContentSize {
   Tight = "tight",
   Medium = "medium",
   Relaxed = "relaxed",
+  Loose = "loose",
```

Comment:
You didn't have to do these updates (could have achieved the same result with only CSS classes passed to the `IconButton` component), but I took this opportunity to extend the `IconButton` component itself.